src/EditorFeatures/Test2/IntelliSense/VisualBasicCompletionCommandHandlerTests.vb

```diff
@@ -2471,7 +2471,7 @@ End Class
 "<Extension> Function IEnumerable(Of 'a).ToArray() As 'a()
 
 Anonymous Types:
-    'a is New With { .x As Integer }")
+    'a is New With { Key .x As Integer }")
```

Comment:
> This looks like a breaking change in behavior of the API, defaults are changed.

Yes.  Intentional and desirable.  Note: the API has not shipped so we are free to change it's behavior.  The API was created specifically for IDE scenarios.  As such, until we actually ship it, we are free to tweak its behavior to best serve the needs of the consumer it was created for :)